contracts/contracts/coordination/AbstractSubscription.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "./Coordinator.sol";
+import "./IFeeModel.sol";
+
+/**
+ * @title Base Subscription contract
+ * @notice Manages the subscription information for rituals.
+ */
+abstract contract AbstractSubscription is IFeeModel {
+    Coordinator public immutable coordinator;
+
+    uint32 public immutable packageDuration;
+    uint32 public immutable yellowPeriodDuration;
+    uint32 public immutable redPeriodDuration;
+
+    /**
+     * @notice Sets subscription parameters
+     * @dev The coordinator and fee token contracts cannot be zero addresses
+     * @param _coordinator The address of the coordinator contract
+     * @param _packageDuration Maximum duration of ritual
```

Comment:
```suggestion
     * @param _packageDuration Maximum duration of first subscription period
```

Reply:
hmm, it's duration of any subscription period, no?

Reply:
Right, maybe `@param _packageDuration Maximum duration of subscription period`? In that case, should the name of the parameter be changed to something like `subscriptionPeriodDuration`?

Reply:
✔️